source/extensions/filters/http/oauth/oauth_client.cc

```diff
@@ -0,0 +1,148 @@
+#include "extensions/filters/http/oauth/oauth_client.h"
+
+#include <chrono>
+
+#include "envoy/http/async_client.h"
+#include "envoy/http/message.h"
+#include "envoy/upstream/cluster_manager.h"
+
+#include "common/common/fmt.h"
+#include "common/common/logger.h"
+#include "common/http/message_impl.h"
+#include "common/json/json_loader.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Oauth {
+
+static const std::string AuthMeApiEndpoint = "/api/me/";
```

Comment:
NO global static `std::string` please, this is source of test flake.
use `constexpr absl::string_view` instead.

Reply:
what is `/api/me/` ? is this an api that the filter expects the server to implement?